Pocket/Pocket/Models/Query.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Query.swift
+//  Pocket
+//
+//  Created by Pabel Nunez Landestoy on 6/5/18.
+//  Copyright © 2018 Pocket Network. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+public class Query: PocketRequestProtocol {
+    public var network = ""
+    public var query = Dictionary<AnyHashable, Any>()
+    public var decoder = Dictionary<AnyHashable, Any>()
+    
+    public func toJSON() -> JSON {
+        return JSON.init(self)
```

Comment:
How do we know that the JSON will only have the 3 keys we want: `network`, `query` and `decoder`.

Reply:
Rip out all of this SwiftJSON stuff and make use of the `Codable` paradigm that I mentioned earlier.